base_tier_validation/tests/test_tier_validation.py

```diff
@@ -155,7 +155,14 @@ def test_08_search_validated(self):
     def test_09_dummy_tier_definition(self):
         """Test tier.definition methods."""
         res = self.tier_def_obj._get_tier_validation_model_names()
-        self.assertEqual(res, [])
+        expected_tier_validation_model_name = []
+        module_id = self.env.ref("base.module_sale_tier_validation")
```

Comment:
You can add raise_if_not_found=False and check if returns something or None
```suggestion
        is_installed = bool(self.env.ref("base.module_sale_tier_validation", raise_if_not_found=False))
```